.github/workflows/ci.yml

```diff
@@ -137,6 +140,7 @@ jobs:
         with:
           toolchain: ${{ env.nightly }}
           override: true
+      - uses: Swatinem/rust-cache@v1
```

Comment:
It doesn't sound like this is useful for the tests that use nightly?

Reply:
Why wouldn't it? It can cache nightly and stable builds

Reply:
The nightly cache would usually be out of date, since it changes every night. But I guess that's ok.